### ISSUE TITLE
Barre d'avancement

### ISSUE THREAD

**Issue by JPPinon** [wecitizens/electoral-gps-frontend#39]
Serait-il possible que les étapes "Stats" et "Résultats" ne deviennent cliquable qu'à partir du moment où l'utilisateur a répondu à minimum 10 questions.

**Comment by olivier5741** [comment]
Yes, I would implement this at the end because it ease the testing process to be able to see the results after giving one answer only. I don't know if manually rewriting the url for testing purposes would give the same result ...

**Comment by JPPinon** [comment]
OK pour attendre la fin des tests. Ce n'est pas un MUST pour le 3/10.
